cmd/test.go

```diff
@@ -55,18 +49,16 @@ https://docs.snyk.io/products/snyk-infrastructure-as-code/custom-rules/getting-s
 		return nil
 	},
 	PostRun: func(cmd *cobra.Command, args []string) {
-		var path string
 		if len(args) == 0 {
-			path = "."
-		} else {
-			path = args[0]
+			// add default path if not provided
+			args = append(args, "rules/", "lib/")
 		}
-		util.CheckIfRunningInRootDirectory(path)
+		util.CheckIfRunningInRootDirectory(args)
 	},
 	RunE: func(cmd *cobra.Command, args []string) error {
 		if len(args) == 0 {
 			// add default path if not provided
```

Comment:
[This comment](https://github.com/snyk/snyk-iac-rules/pull/96/files#r757895741) also applies here.